cmd/crc/cmd/status.go

```diff
@@ -79,25 +79,41 @@ func runWatchStatus(writer io.Writer, client *daemonclient.Client, cacheDir stri
 		ramBar  *pb.ProgressBar
 	)
 
+	var err error
+	barPull = pb.NewPool()
```

Comment:
Can you use this opportunity to rename `barPull` to `barPool`? I think this is a typo.